taipo/common.py

```diff
@@ -18,6 +18,7 @@ def nlu_path_to_dataframe(path):
     res = (
         Clumper.read_yaml(path)
         .explode("nlu")
+        .keep(lambda d: d["nlu"] if ("intent" in d["nlu"].keys()) else False)
```

Comment:
I'll hit merge in a moment for speed, but `.keep()` accepts a function that returns `True` or `False` for every item in the collection. If there's an element that is "truthy" then it will keep it. 

That means that this would suffice;

```python
.keep(lambda d: "intent" in d["nlu"].keys())
```

If you're new to this "functional way of coding", you might appreciate the precursor series on [method chaining on calmcode](https://calmcode.io/method-chains/introduction.html).

Reply:
This is better! Thanks for the insights!